crates/viewer/re_viewport_blueprint/src/container.rs

```diff
@@ -79,15 +79,15 @@ impl ContainerBlueprint {
         // This is a required component. Note that when loading containers we crawl the subtree and so
         // cleared empty container paths may exist transiently. The fact that they have an empty container_kind
         // is the marker that the have been cleared and not an error.
-        let container_kind = results.component_instance::<ContainerKind>(0)?;
+        let container_kind = results.component_mono::<ContainerKind>()?;
```

Comment:
Can we use `component_instance` and `component_mono` interchangeably?

Reply:
IIRC the difference is that some (?) `component_mono` implementations also will check whether a component was truly mono and allow you to issue a warning